blog/breach-to-barrier.md

```diff
@@ -0,0 +1,33 @@
+---
+title: 'Breach to Barrier: Strengthening Apps with the Sandbox'
+date: 2023-09-24T00:00:00.000Z
+authors:
+  - name: felixrieseberg
+    url: 'https://github.com/felixrieseberg'
+    image_url: 'https://github.com/felixrieseberg.png?size=96'
+slug: breach-to-barrier
+---
+
+It’s been more than a week since [CVE-2023-4863: Heap buffer overflow in WebP](https://chromereleases.googleblog.com/2023/09/stable-channel-update-for-desktop_11.html) was made public, leading to a flurry of new releases of software rendering `webp` images: macOS, iOS, Chrome, Firefox, and various Linux distributions all received updates. This followed investigations by Citizen Lab, discovering that an iPhone used by a “Washington DC-based civil society organization” was under attack using a zero-click attack within iMessage.
+
+Electron, too, spun into action and released new versions the same day: If your app renders any user-provided content, you should update your version of Electron - v27.0.0-beta.2, v26.2.1, v25.8.1, v24.8.3, and v22.3.24 all contain a fixed version of `libwebp`, the library responsible for rendering `webp` images.
+
+Now that we are all freshly aware that an interaction as innocent as “rendering an image” is a potentially dangerous activity, we want to use this opportunity to remind everyone that Electron comes with a sandbox that will limit the blast radius of the next big attack — whatever it may be.
+
+The memory sandbox was available ever since Electron v1 and enabled by default in v20, but we know that many apps (especially those that have been around for a while) may have a `sandbox: false` somewhere in their code. That’s understandable: If you’ve been with us for a long time, you probably enjoyed the power of throwing a `require("child_process")` or `require("fs")` into the same code that runs your HTML/CSS.
```

Comment:
Nitpick: technically sandbox is disabled if you enable nodeIntegration (or was it disabling context isolation?). Dunno how to fit that fact in without sounding wordy. Maybe a footnote?